CHANGELOG.md

```diff
@@ -9,9 +9,12 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Added
 
-### Changed
 
-### Deleted
+## [1.6.2] - 2024-05-30
```

Comment:
why do you add this section title ?
please remove it and let your new line added under "Unreleased" section